docs/Options.md

```diff
@@ -266,6 +266,18 @@ Ruby section is shown only in directories that contain `Gemfile`, or `Rakefile`,
 | `SPACESHIP_RUBY_SYMBOL` | `💎·` | Character to be shown before Ruby version |
 | `SPACESHIP_RUBY_COLOR` | `red` | Color of Ruby section |
 
+### Python (`python`)
+
+Python version is shown only inside directories that contain relevant files like `requirements.txt`, `Pipfile` or any other file with `.py` extension with virtual environment activated or if section is configured to be displayed `always` with relevant files.
```

Comment:
Should also add the `pyproject.toml` file to the relevant files.

Reply:
Done :+1: